integration_tests/inttest_lib/runners/lambda_function_runner.py

```diff
@@ -74,8 +75,10 @@ def _is_lambda_completed(self, lambda_function_name, request_id, start_time):
         query_string = f"fields @timestamp, @message | filter @message like /END RequestId: {request_id}/"
 
         try:
+            query_start_time = int(datetime_to_epoch_milliseconds(start_time))
```

Comment:
since we are relying on int, which is epoch msec, I'd rather used int(time.time()*1000) directly as it provides UTC, tz-aware value.
maybe, we can change start_time assignment in await_completion method.
Can discuss during the next sync.

p.s.: removed my previous comment as it didn't make sense :)
but this, hopefully, does

Reply:
thank you for your comments!
sure :) let me try to check on that as well